### DISCUSSION TITLE
About Results from utils package

### DISCUSSION THREAD

**Discussion by anooppandey276** [PMEAL/porespy#832] [General]
Hey everyone, I was trying to import Results from ._utils but I am facing problem and got an error
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Input In [32], in <cell line: 1>()
----> 1 from .utils import Results

ImportError: attempted relative import with no known parent package

Can anybody let me know, how to fix it

**Comment by jgostick** [comment]
This usually happens when you are running a file from a strange location, like within the porespy directory.
